doc/classes/NetworkedMultiplayerPeer.xml

```diff
@@ -13,31 +15,35 @@
 			<return type="int" enum="NetworkedMultiplayerPeer.ConnectionStatus">
 			</return>
 			<description>
+				Returns the current state of the connection. See [enum ConnectionStatus].
 			</description>
 		</method>
 		<method name="get_packet_peer" qualifiers="const">
 			<return type="int">
 			</return>
 			<description>
+				Returns the ID of the [class NetworkedMultiplayerPeer] who sent the most recent packet.
```

Comment:
Class hyperlinks are not done with `[class ClassName]`, but directly with `[ClassName]`.

Though in this case, we don't want looping references to self, so you should use `[code]NetworkedMultiplayerPeer[/code]` to avoid unnecessary hyperlinks.